kickstart/provision_rhel_atomic.erb

```diff
@@ -0,0 +1,27 @@
+<%#
+kind: provision
+name: RHEL Atomic Kickstart
```

Comment:
Is this RHEL-specific or is it, like the oses list above, applicable to CentOS and Fedora?